includes/class-sensei-quiz.php

```diff
@@ -2417,6 +2424,48 @@ public static function is_quiz_awaiting_grade_for_user( $lesson_id = null, $user
 
 		return $lesson_status && 'ungraded' === $lesson_status->comment_approved;
 	}
+
+	/**
+	 * Returns next lesson URL when meets the conditions of the next lesson button to be displayed
+	 * in the quiz footer, returns null otherwise.
+	 *
+	 * @param ?int $lesson_id The lesson ID.
+	 * @param ?int $user_id   The user ID.
+	 *
+	 * @return string|null Next lesson URL if condition holds, null otherwise.
+	 */
+	public static function maybe_get_next_lesson_url_for_quiz_footer( $lesson_id = null, $user_id = null ) {
```

Comment:
Could this be `private`?

Reply:
Made it private here https://github.com/Automattic/sensei/pull/7202/commits/57e56f927b042edb94a8f823f4d18a048597eab6